zookeeper-server/src/main/java/org/apache/zookeeper/common/X509Util.java

```diff
@@ -61,6 +61,16 @@
 public abstract class X509Util {
     private static final Logger LOG = LoggerFactory.getLogger(X509Util.class);
 
+    static {
+        // Client-initiated renegotiation in TLS is unsafe and
+        // allows MITM attacks, so we should disable it unless
+        // it was explicitly enabled by the user.
+        String propertyName = "jdk.tls.rejectClientInitiatedRenegotiation";
+        if (System.getProperty(propertyName) == null) {
+            System.setProperty(propertyName, Boolean.TRUE.toString());
```

Comment:
Add a log to show disabling this to make it easier for people to aware this is being disabled in ZK by default.

Reply:
Done